datanode/gateway/graphql/schema.graphql

```diff
@@ -144,6 +144,14 @@ type Subscription {
     "the party ID to subscribe for, empty if all"
     partyId: ID
   ): Reward!
+
+  "Subscribe to liquidity provisioning data"
+    liquidityProvisions(
+        "the party ID to subscribe for, empty if all"
+        partyId: ID
+        "the market ID to subscribe for, empty if all"
+        marketId: ID
+    ): [LiquidityProvision!]
```

Comment:
This should be a specific type flattened like for other subscribtions (e.g: orders, trades etc).

Reply:
This is like this because of batching. The orders, trades subscriptions are declared:

```
  "Subscribe to orders updates"
  orders(
    "ID of the market from which you want orders updates"
    marketId: ID
    "ID of the party from which you want orders updates"
    partyId: ID
  ): [Order!]

  "Subscribe to the trades updates"
  trades(
    "ID of the market from which you want trades updates"
    marketId: ID
    "ID of the party from which you want trades updates"
    partyId: ID
  ): [TradeUpdate!]
```

The liquidity provisions events handler in the data-node also batches up the liquidity provision events before inserting to the database, so I've duplicated the subscription to send the batched updates. I can change it to stream them individually each time the batch is processed if you think this would be better?

Reply:
Sorry for the misunderstanding. I've addressed the requirement in https://github.com/vegaprotocol/vega/pull/6102/commits/04543f998b4fc67bb19ff712508089cfcf297e93